scripts/drone/steps/lib.star

```diff
@@ -567,7 +567,7 @@ def test_backend_step(image = images["build_image"]):
             "wire-install",
         ],
         "commands": [
-            "go test -tags requires_buildifer -short -covermode=atomic -timeout=5m ./pkg/...",
+            "make test-go-unit",
```

Comment:
Hmm... do we really need a 30m timeout in the makefile?

Reply:
wanted to play it safe and keep the bigger of the two but with 5m it works fine. Should I use that instead?

Reply:
If that works for you, then please :) Usually, if the unittests aren't done within 5 minutes, there is something wrong anyway :D

Reply:
Thank you :)